content/02.body.md

```diff
@@ -1,13 +1,23 @@
 ## Introduction
 
-Networks contain information about relationships between entities ("edges between nodes").
-A node's degree is the number of relationships it has in the network.
+Networks contain information about relationships between entities (referred to here as "edges" between "nodes").
+A node's degree is the number of edges it has in the network.
+`TODO: do we need to discuss in-degree vs out-degree and hetnet degree here?`
 Networks contain many nodes, whose degrees can be aggregated to form the network's degree distribution.
-Because different nodes can have very different degrees, real networks have highly variable degree distributions (Figure {@fig:hetionet}).
+Because different nodes can have very different degrees, real networks generally have highly variable degree distributions [@doi:10.1126/science.286.5439.509; @doi:10.1038/s41467-019-08746-5; @doi:10.1371/journal.pone.0017645].
+This is especially true for networks encoding biomedical knowledge or assays — where natural forces such as preferential attachment inherent to the problem domain combine with observation-based influences such as study methodology — to create non-uniform degree distributions (Figure {@fig:hetionet}).
 
-![Degree distribution can vary greatly between networks, even between the source and target degree distributions within the same (directed or bipartite) network.
-  Shown are seven degree distributions for six edge type subnetworks of Hetionet [@doi:10.7554/eLife.26726].
-  (The final two are the source and target distributions of the gene-regulates-gene directed edge type.)
+![
+**Biomedical networks are characterized by non-uniform degree distributions.**
+Seven degree distributions are plotted for six edge types Hetionet v1.0 [@doi:10.7554/eLife.26726].
+Hetionet integrates subnetworks for 24 different edge types.
+To assess the degree distributions of Hetionet, each edge type is analyzed separately.
+Furthermore, bipartite (e.g. Anatomy→expresses→Gene) and directed (e.g. Gene→regulates→Gene) edge types have both source and target degrees that must be assessed separately.
+Undirected edge types (e.g Compound–resembles–Compound) have only a single degree distribution.
+Degree distributions are non-uniform and vary greatly between different networks.
+The y-axis is XX-scaled to accomodate the common occurrence where most nodes have low degree while a small portion of nodes have high degree.
```

Comment:
is this log scaled? I wonder if that is how the zero-degrees are filtered? Also is scaled rather than transformed the proper term as per the grammar of graphiques?

Reply:
The [notebook where this figure was made](https://github.com/greenelab/xswap-analysis/blob/master/nb/3.fig1.hetionet_degree_dist/2.plot_hetionet_degree_distributions.ipynb) shows a warning message from ggplot that, "Transformation introduced infinite values in continuous y-axis." These rows (zero degree nodes) were dropped from the visualization.

Scaling is log10.

```suggestion
The y-axis is log-10-scaled to accomodate the common occurrence where most nodes have low degree while a small portion of nodes have high degree.
```